pkg/restapi/operation/operations.go

```diff
@@ -14,6 +14,8 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/trustbloc/hub-auth/pkg/bootstrap/user"
+
```

Comment:
Move this import down with the other `hub-auth` import

Reply:
fixed